sql/report_util_functions.sql

```diff
@@ -345,9 +345,7 @@ CREATE FUNCTION patientPickedARVDrugDuringReportingPeriod(
     p_protocolLineNumber INT(11)) RETURNS TINYINT(1)
     DETERMINISTIC
 BEGIN
-
```

Comment:
@IipperMangesh I would like to suggest we keep this space (and on line 350 as well), this will make the following SQL code bloc separated and more readable